StRoot/StEvent/StFttPoint.cxx

```diff
@@ -0,0 +1,20 @@
+/***************************************************************************
+ *
+ * StFttPoint.cxx
+ *
+ * Author: jdb 2021
+ ***************************************************************************
+ *
+ * Description: Implementation of StFttPoint, the StEvent point structure
+ *
+ ***************************************************************************/
+#include "StFttPoint.h"
+#include "St_base/StMessMgr.h"
+
+
+StFttPoint::StFttPoint() :  StObject() { /* no op */ }
+
+StFttPoint::~StFttPoint() { /* no op */ }
```

Comment:
A minor comment on the performance side. If the destructor does nothing, there is no need to define an empty one. If we do, we may prevent the compiler to generate the move and move assignment constructors under C++11. In that case, we lost some optimization opportunities.